hotel_calendar/static/src/js/views/calendar/hotel_calendar_controller.js

```diff
@@ -634,11 +635,23 @@ var PMSCalendarController = AbstractController.extend({
 
     _generate_reservation_tooltip_dict: function(tp) {
       return {
+        'folio_name': tp['folio_name'],
         'name': tp['name'],
         'phone': tp['phone'],
-        'arrival_hour': HotelCalendar.toMomentUTC(tp['checkin'], HotelConstants.ODOO_DATETIME_MOMENT_FORMAT).local().format('HH:mm'),
+        'email': tp['email'],
+        'room_type': tp['room_type'],
+        'adults': tp['adults'],
+        'children': tp['children'],
+        'checkin': HotelCalendar.toMomentUTC(tp['checkin'], '').format("DD MMMM"),
+        'checkin_day_of_week': HotelCalendar.toMomentUTC(tp['checkin'], '').format("dddd"),
+        'checkout': HotelCalendar.toMomentUTC(tp['checkout'], '').format("DD MMMM"),
+        'checkout_day_of_week': HotelCalendar.toMomentUTC(tp['checkout'], '').format("dddd"),
```

Comment:
same here... and perhaps can use the same "moment" object to print it in different formats.